api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/AbstractPool.java

```diff
@@ -126,4 +162,11 @@ private boolean isComponentInUse(C component) {
         return true;
     }
 
+    private boolean isInModel(String uri) {
+        Resource s = dynamicAPIModel.getResource(uri);
+        Property p = dynamicAPIModel.getProperty(RDF_TYPE_PROPERTY); 
+
+        return dynamicAPIModel.contains(s, p);
```

Comment:
Is there an option to verify that object type is right (not only subject and predicate)?